src/explore-education-statistics-common/src/styles/_content.scss

```diff
@@ -18,6 +18,10 @@
     max-width: 100%;
     min-width: 50px;
   }
+
```

Comment:
I don't suppose this might cause any Robot tests to fail due to some table cell no longer being visible?